arangod/IResearch/IResearchViewCoordinator.cpp

```diff
@@ -270,7 +272,7 @@ bool IResearchViewCoordinator::emplace(
       irs::string_ref const& key
   )->bool {
     return key != arangodb::StaticStrings::IndexType
-      && key != StaticStrings::ViewIdField; // ignored fields
+        && key != StaticStrings::ViewIdField; // ignored fields
```

Comment:
the view ID field is required, this is not the correct place for the filter, the correct place is in appendVelocyPackDetailed(...)

Reply:
As you may notice, I only changed the formatting here. It was already masked out previously. Adding a mask here also to remove `arangodb::StaticStrings::IndexId` is fine in the cluster case, it causes failing tests in single server.